src/main/java/de/theit/jenkins/crowd/CrowdConfigurationService.java

```diff
@@ -558,10 +590,16 @@ public boolean isUserNestedGroupMember(String username, String groupname)
         }
     }
 
-    public void validateSSOAuthentication(String token, List<ValidationFactor> list)
-            throws OperationFailedException, InvalidAuthenticationException, ApplicationPermissionException,
-            InvalidTokenException {
-        LOG.log(Level.FINEST, "CrowdClient.validateSSOAuthentication()");
+    public void validateSSOAuthentication(String token, List<ValidationFactor> list) throws OperationFailedException, InvalidAuthenticationException, ApplicationPermissionException, InvalidTokenException {
+        // Load the entry from cache
+        // if it's located in cache this means it was already validated
+        Boolean retval = getValidValueFromCache(token, validationCache);
+        if (retval != null) {
+            LOG.log(Level.FINEST, "validateSSOAuthentication() cache hit: {0}...", token.substring(0,12));
+            return;
+        }
+
+        LOG.log(Level.FINEST, "validateSSOAuthentication() cache hit MISS: {0}...", token.substring(0,12));
```

Comment:
Please remove dumping toknes.
Please change this log to debug. Mostly because it could be pretty extensive in bigger organization.

Reply:
Removed token dumping.